src/main/resources/application.properties

```diff
@@ -8,3 +8,20 @@ quarkus.resteasy.path=/api/v1
 #quarkus.log.level=DEBUG
 quarkus.application.version=1.0.0-SNAPSHOT
 quarkus.locales=en-US
+
+# Backend type (inmemory or vault)
+spi.backend.type=inmemory
+
+# Uncomment for your Vault settings:
+
+# vault url
```

Comment:
do they are mandatory for the unit tests?

Reply:
looks like no